src/vs/platform/workspace/common/workspace.ts

```diff
@@ -120,17 +132,18 @@ export class WorkspaceContextService implements IWorkspaceContextService {
 		return null;
 	}
 
-	public tryGetWorkspaceFromRegistry(resource: URI): IWorkspaceRevState | undefined {
+	public tryGetWorkspaceFromRegistry(resource: URI): IWorkspace | undefined {
 		return this.workspaceRegistry.get(resource.toString());
```

Comment:
Doesn't this need to do the same `.with({ fragment: '', query: '' })` as above/below?